src/baseline.rs

```diff
@@ -244,25 +255,54 @@ impl BaselineLoader for RegistryBaseline {
         ));
         std::fs::create_dir_all(&base_root)?;
         let manifest_path = base_root.join("Cargo.toml");
-        std::fs::write(
-            &manifest_path,
-            format!(
-                "
-[workspace]
-
-[package]
-name = 'rustdoc'
-version = '0.0.0'
-publish = false
 
-[lib]
-path = 'lib.rs'
+        let manifest = {
+            use cargo_toml::*;
+            // tonowak to obi1kenobi: is there an easy way of using cargo_toml::*, but without
+            // bringing it to the scope of the whole function? With those nested let I've managed
+            // to not pollute the scope, but it seems stupid.
```

Comment:
Consider extracting the manifest-construction step into its own function which can have `cargo_toml::*` in scope just for itself?

Reply:
Now that I think about it, I can just return the constructed Manifest, without saving it in the inner variable (leaving the `use cargo_toml::*` as it is). Do you consider it to be cleaner than extracting it to a separate function?

Reply:
I don't think I understood what you were describing.

In general, this function is quite long and complex so I think it might be worth extracting the manifest-building to its own function regardless of the `cargo_toml::*` import.

Reply:
> I am short on sleep and under-caffeinated this morning

We're not in a hurry. If you need some rest, we won't mind waiting for a day or for a few days :)

> In general, this function is quite long and complex so I think it might be worth extracting the manifest-building to its own function regardless of the cargo_toml::* import.

Good idea, I'll do that.

Reply:
> We're not in a hurry. If you need some rest, we won't mind waiting for a day or for a few days :)

Thanks, I appreciate it. But it's all good, it's just hard to explain, understand, and visualize changes to code described in human languages -- or else we'd just be having conversations with the compiler instead of writing code :)